glustercli/cmd/tracing.go

```diff
@@ -0,0 +1,183 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	tracemgmtapi "github.com/gluster/glusterd2/plugins/tracemgmt/api"
+
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	helpTraceCmd            = "Gluster Trace Management"
+	helpTraceEnableCmd      = "Enable Tracing"
+	helpTraceStatusCmd      = "Show Tracing Status"
+	helpTraceUpdateCmd      = "Update Tracing Options"
+	helpTraceDisableCmd     = "Disable Tracing"
+	errTraceEnableReqFailed = "Failed to enable tracing"
+	errTraceUpdateReqFailed = "Failed to update trace configuration"
+)
+
+var (
+	flagTraceJaegerEndpoint       string
+	flagTraceJaegerAgentEndpoint  string
+	flagTraceJaegerSampler        int
+	flagTraceJaegerSampleFraction float64
+)
+
+func init() {
+	// Trace Enable
+	traceEnableCmd.Flags().IntVar(&flagTraceJaegerSampler, "jaeger-sampler", 2, "jaeger sampler (1: Always sample, 2: Probabilistic)")
+	traceEnableCmd.Flags().Float64Var(&flagTraceJaegerSampleFraction, "jaeger-sample-fraction", 0.1, "jaeger sample fraction (min: >0.0, max: <1.0)")
+	// Trace Update
+	traceUpdateCmd.Flags().StringVar(&flagTraceJaegerEndpoint, "jaeger-endpoint", "", "jaeger endpoint")
+	traceUpdateCmd.Flags().StringVar(&flagTraceJaegerAgentEndpoint, "jaeger-agent-endpoint", "", "jaeger agent endpoint")
+	traceUpdateCmd.Flags().IntVar(&flagTraceJaegerSampler, "jaeger-sampler", 2, "jaeger sampler (1: Always sample, 2: Probabilistic)")
+	traceUpdateCmd.Flags().Float64Var(&flagTraceJaegerSampleFraction, "jaeger-sample-fraction", 0.1, "jaeger sample fraction (min: >0.0, max: <1.0)")
+	traceCmd.AddCommand(traceEnableCmd)
+	traceCmd.AddCommand(traceStatusCmd)
+	traceCmd.AddCommand(traceUpdateCmd)
+	traceCmd.AddCommand(traceDisableCmd)
+}
+
+var traceCmd = &cobra.Command{
+	Use:   "trace",
+	Short: helpTraceCmd,
+}
+
+var traceEnableCmd = &cobra.Command{
+	Use:   "enable <jaeger-endpoint> <jaeger-agent-endpoint>",
+	Short: helpTraceEnableCmd,
+	Args:  cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		jaegerEndpoint := args[0]
+		jaegerAgentEndpoint := args[1]
+
+		if jaegerEndpoint == "" || jaegerAgentEndpoint == "" {
+			err := errors.New("One or more Jaeger endpoints not specified")
```

Comment:
error message should start with small case , please update in other places also

Reply:
Updated here and in all other places as well.